tests/integration/test_remoted/conftest.py

```diff
@@ -23,3 +25,26 @@ def restart_remoted(get_configuration, request):
         control_service('start', daemon=DAEMON_NAME)
     except sb.CalledProcessError:
         pass
+
+
+@pytest.fixture(scope="module")
+def create_agent_group():
+    """Temporary creates a new agent group for testing purpose, must be run only on Managers."""
+
+    subprocess.run(["/var/ossec/bin/agent_groups", "-q", "-a", "-g", "testing_group"])
+
+    with open("/var/ossec/etc/shared/testing_group/agent.conf", "w") as f:
+        f.write('''
+                <!-- custom configuration file
+                    example for agents! -->
+                <agent_config>
+                    <localfile>
+                        <location>/var/log/my_test.log</location>
+                        <log_format>syslog</log_format>
+                    </localfile>
+                </agent_config>
+                ''')
+
+    yield
+
+    subprocess.run(["/var/ossec/bin/agent_groups", "-q", "-r", "-g", "testing"])
```

Comment:
The module was previously imported as `sb`. You should use that instead. Also, try to use the `WAZUH_PATH` variable from the module instead of using `/var/ossec`

Reply:
https://github.com/wazuh/wazuh-qa/commit/3551aff023ce1af0ae50bcb94d1dd7ff8defa024